source/images/buyers-guide/icon-bubble.svg

```diff
@@ -1,4 +1 @@
-<?xml version="1.0" encoding="utf-8"?>
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 120 38">
-<path fill="#f2f2f2" d="M112.4,23.8c0,6.4,2.5,11.1,7.6,13.9c-7-0.5-11.9-1.8-14.7-3.9c-3.4,2.7-7.6,4.2-11.9,4.2H19C8.5,38,0,29.5,0,19S8.5,0,19,0h74.4c10.5,0,19,8.5,19,19C112.4,19.9,112.4,22.9,112.4,23.8z"/>
-</svg>
+<?xml version="1.0" encoding="utf-8"?><svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="120" height="38" viewBox="0 0 120 38"><path fill="#f2f2f2" d="M112.4,23.8c0,6.4,2.5,11.1,7.6,13.9c-7-0.5-11.9-1.8-14.7-3.9c-3.4,2.7-7.6,4.2-11.9,4.2H19C8.5,38,0,29.5,0,19S8.5,0,19,0h74.4c10.5,0,19,8.5,19,19C112.4,19.9,112.4,22.9,112.4,23.8z"/></svg>
```

Comment:
we might not need this file anymore, but it feels smart to keep it around, just in case?

Reply:
Let's remove the file if we aren't using it at the moment. Design team should have assets in the design files / folders so exporting icons should be fairly easy if we ever need any again.